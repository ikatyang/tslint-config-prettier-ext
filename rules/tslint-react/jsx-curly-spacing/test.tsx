const fail = <App foo={bar } />;
