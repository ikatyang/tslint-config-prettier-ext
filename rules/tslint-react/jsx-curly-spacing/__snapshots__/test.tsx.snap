// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const fail = <App foo={bar } />;

======
const fail = <App foo={bar} />;

>>>>>> after
"
`;

exports[`should fail before formatting 1`] = `
"ERROR: (jsx-curly-spacing) /rules/tslint-react/jsx-curly-spacing/test.tsx[1, 28]: There should be no space before }
"
`;
